test-engineering/load-tests/setup_k8.sh

```diff
@@ -0,0 +1,110 @@
+    $GCLOUD container images list | grep locust-merino
+
+    ##Deploying the Locust master and worker nodes
+    echo -e "==================== Update Kubernetes Manifests "
+    echo -e "==================== Replace the target host and project ID with the deployed endpoint and project ID in the locust-master-controller.yml and locust-worker-controller.yml files"
+
+    FILES=($MASTER_FILE $WORKER_FILE)
+    for file in "${FILES[@]}"
+    do
+        $SED -i -e "s|\[TARGET_HOST\]|$TARGET|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[PROJECT_ID\]|$GOOGLE_CLOUD_PROJECT|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[LOCUST_IMAGE_TAG\]|$LOCUST_IMAGE_TAG|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[KINTO__BUCKET\]|$KINTO__BUCKET|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[KINTO__COLLECTION\]|$KINTO__COLLECTION|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[KINTO__SERVER_URL\]|$KINTO__SERVER_URL|g" $MERINO_DIRECTORY/$file
+        $SED -i -e "s|\[WORKER_COUNT\]|$WORKER_COUNT|g" $MERINO_DIRECTORY/$file
+
+    done
+
+    ##Deploy the Locust master and worker nodes using Kubernetes Manifests
+    echo -e "==================== Deploy the Locust master and worker nodes"
+    $KUBECTL apply -f $MERINO_DIRECTORY/$MASTER_FILE
+    $KUBECTL apply -f $MERINO_DIRECTORY/$SERVICE_FILE
+    $KUBECTL apply -f $MERINO_DIRECTORY/$WORKER_FILE
+
+    echo -e "==================== Verify the Locust deployments & Services"
+    $KUBECTL get pods -o wide
+    $KUBECTL get services
+}
+
+echo "==================== The script is used to create & delete the GKE cluster"
+echo "==================== Do you want to create or setup the existing or delete GKE cluster? Select ${BOLD}create or delete or setup ${NORM}"
+while :
+do
+    read response
+    case $response in
+        create) #Setup Kubernetes Cluster
+            echo -e "==================== Creating the GKE cluster "
+            $GCLOUD container clusters create $CLUSTER --region $REGION --scopes $SCOPE --enable-autoscaling --min-nodes "10" --max-nodes "15" --scopes=logging-write,storage-ro --addons HorizontalPodAutoscaling,HttpLoadBalancing  --machine-type $MACHINE_TYPE
+            SetupGksCluster
+            #CREDS=$($GCLOUD container clusters get-credentials $CLUSTER --region $REGION --project $GOOGLE_CLOUD_PROJECT)
+            #echo -e "==================== To Connect to the GKE cluster use: $CREDS "
+            break
+            ;;
+        delete)
+            echo -e "==================== Delete the GKE cluster "
+            $GCLOUD container clusters delete $CLUSTER --region $REGION
+            break
+            ;;
+        setup)
+            echo -e "==================== Setup the GKE cluster "
+            SetupGksCluster
+            break
+            ;;
+        *)
+            echo -e "==================== Incorrect input! "
+            break
+            ;;
+    esac
+done
```

Comment:
Missing a newline character at the end of the file.

```suggestion
done

```